subprojects/docs/src/docs/userguide/dep-man/06-publishing/publishing_maven.adoc

```diff
@@ -188,6 +188,54 @@ For integration with a local Maven installation, it is sometimes useful to publi
 
 The Maven Publish Plugin makes this easy to do by automatically creating a link:{javadocPath}/org/gradle/api/publish/maven/tasks/PublishToMavenLocal.html[PublishToMavenLocal] task for each link:{groovyDslPath}/org.gradle.api.publish.maven.MavenPublication.html[MavenPublication] in the `publishing.publications` container. The task name follows the pattern of `publish__PubName__PublicationToMavenLocal`. Each of these tasks is wired into the `publishToMavenLocal` aggregate task. You do not need to have `mavenLocal()` in your `publishing.repositories` section.
 
+[[publishing_maven:relocation]]
+== Publishing Maven relocation information
+
+When a project changes the `groupId` or `artifactId` (the _coordinates_) of an artifact it publishes, it is important to let users know where the new artifact can be found. Maven can help with that through the _relocation_ feature. The way this works is that a project publishes an additional artifact under the old coordinates consisting only of a minimal _relocation POM_; that POM file specifies where the new artifact can be found. Maven repository browsers and build tools can then inform the user that the coordinates of an artifact have changed.
```

Comment:
Technically Maven coordinates consist of groupId, artifactId **and version**, though maybe it is acceptable here to call only groupId and artifactId "coordinates" because that makes referring to them easier in the following sections.

Reply:
This seems fine.